Directory.Packages.props

```diff
@@ -0,0 +1,147 @@
+  </PropertyGroup>
+  <ItemGroup>
+    <!-- Note: These libraries can be loaded by Arcade SDK clients, which may already have a
+         newer copy of this package loaded. When updating one, be aware of the other to prevent runtime issues
+         See: https://github.com/dotnet/arcade/blob/main/eng/Versions.props -->
+    <PackageVersion Include="Azure.Core" Version="1.20.0" />
+    <PackageVersion Include="Azure.Data.AppConfiguration" Version="1.0.0" />
+    <PackageVersion Include="Azure.Data.Tables" Version="12.2.0" />
+    <PackageVersion Include="Azure.Identity" Version="1.5.0" /> 
+    <PackageVersion Include="Azure.Security.KeyVault.Keys" Version="4.0.1" />
+    <PackageVersion Include="Azure.Security.KeyVault.Secrets" Version="4.0.1" />
+    <PackageVersion Include="Azure.Storage.Blobs" Version="12.3.0" />
+    <PackageVersion Include="Azure.Storage.Queues" Version="12.6.0" />
+    <PackageVersion Include="Castle.Core" Version="4.3.0" />
+    <PackageVersion Include="CommandLineParser" Version="2.2.1" />
+    <PackageVersion Include="EntityFrameworkCore.Triggers" Version="1.1.1" />
+    <PackageVersion Include="FluentAssertions" Version="5.10.3" />
+    <PackageVersion Include="FluentValidation.AspNetCore" Version="8.6.2" />
+    <PackageVersion Include="FluentAssertions.Json" Version="5.5.0" />
+    <PackageVersion Include="Humanizer.Core" Version="2.8.26" />
+    <PackageVersion Include="JetBrains.Annotations" Version="11.1.0" PrivateAssets="All" />
+    <PackageVersion Include="LibGit2Sharp" Version="0.27.0-preview-0175" />
+    <PackageVersion Include="Microsoft.ApplicationInsights" Version="2.18.0" />
+    <PackageVersion Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.18.0" />
+    <PackageVersion Include="Microsoft.ApplicationInsights.DependencyCollector" Version="2.18.0" />
+    <PackageVersion Include="Microsoft.ApplicationInsights.PerfCounterCollector" Version="2.18.0" />
+    <PackageVersion Include="Microsoft.ApplicationInsights.ServiceFabric" Version="2.3.1" />
+    <PackageVersion Include="Microsoft.AspNet.WebApi.Client" Version="5.2.6" />
+    <PackageVersion Include="Microsoft.AspNetCore.Authentication" Version="2.2.0" />
+    <PackageVersion Include="Microsoft.AspNetCore.DataProtection.AzureKeyVault" Version="$(AspNetAzureVersion)" />
+    <PackageVersion Include="Microsoft.AspNetCore.DataProtection.AzureStorage" Version="$(AspNetAzureVersion)" />
+    <!-- Force version 2.1.22 to avoid unsafe versions -->
+    <PackageVersion Include="Microsoft.AspNetCore.Http" Version="2.2.0" />
+    <PackageVersion Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="$(AspNetCoreVersion)" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="$(AspNetCoreVersion)" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="$(AspNetCoreVersion)" />
+    <!-- Force version 2.0.2 to avoid unsafe versions -->
```

Comment:
Is this a transitive thing? Or is basically _everything_ transitively pinned?

Reply:
Anything that shows up in this file, If it is included transitively, will be fixed to the version in this file.

Reply:
This comment isn't one I added though, it was in the Packages.props. This file is mostly a rename of that one tweaked to use the new stuff.